config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  post '/event_handler', to: 'webhook#handle'
```

Comment:
will this route be just for github or could it be by something else in the future? asking because if it's tightly coupled to github then I'd make it clear in the route